source/sass/patterns/organisms/site-header.scss

```diff
@@ -1,20 +1,33 @@
-@import "../../mixins/grid";
 @import "../../mixins/media-query";
 @import "../../mixins/sizes";
 @import "../../mixins/spacing";
 
 .site-header {
-  @include margin(6px auto 0);
-  @include max-inline-size($grid-max_width);
+  display: flex;
+  @include min-block-size(59px);
+  @include block-spacing($start: 8px, $end: 7px);
+  @include margin(18px, inline);
+  align-items: center;
+
+  @include mq($from: wide) {
+    @include min-block-size(95px);
+  }
+
+  &.content-grid__item {
```

Comment:
Feels like this shouldn't be in a `content-grid`. May not even a sub-grid at all?